src/packet.c

```diff
@@ -1267,7 +1267,7 @@ _libssh2_packet_require(LIBSSH2_SESSION * session, unsigned char packet_type,
         }
         else if(ret == 0) {
             /* nothing available, wait until data arrives or we time out */
-            long left = LIBSSH2_READ_TIMEOUT - (long)(time(NULL) -
+            long left = session->packet_read_timeout - (long)(time(NULL) -
                                                       state->start);
```

Comment:
and here too.

Reply:
We must have been asleep at the indentation wheel, fixed this one too, thank you.